README.md

```diff
@@ -72,6 +72,12 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](ruthmy.github.io)
```

Comment:
- [ ] Well done deploying your project and updating your readme file 👏🏼 However the link is not complete and to make it works I recommend adding the full path so it looks something like this ⬇️ 

```
https://ruthmy.github.io/portfolio/
```

**OR**

```
ruthmy.github.io/portfolio/
```